src/Raven.Server/Smuggler/Documents/Data/DatabaseSmugglerOptionsServerSide.cs

```diff
@@ -8,21 +8,29 @@ namespace Raven.Server.Smuggler.Documents.Data
 {
     public sealed class DatabaseSmugglerOptionsServerSide : DatabaseSmugglerOptions, IDatabaseSmugglerImportOptions, IDatabaseSmugglerExportOptions
     {
+        private DatabaseSmugglerOptionsServerSide()
+        {
+            //Only for GenerateJsonDeserializationRoutine
+        }
+        public DatabaseSmugglerOptionsServerSide(AuthorizationStatus authorizationStatus)
+        {
+            AuthorizationStatus = authorizationStatus;
+        }
         public bool ReadLegacyEtag { get; set; }
 
         public string FileName { get; set; }
 
-        public AuthorizationStatus AuthorizationStatus { get; set; } = AuthorizationStatus.ValidUser;
+        public AuthorizationStatus AuthorizationStatus { get; set; }
```

Comment:
Can we make it a private `set;` here? In order to force setting it via constructor

Reply:
Done